### DISCUSSION TITLE
How to migrate the settings to a different computer?

### DISCUSSION THREAD

**Discussion by xonline** [Marekkon5/onetagger#221] [Q&A] [answered]
I got 2 computers and under quicktag I have lots of customised tags. I don't want to having to redo it again on a different computer. is there a way to copy what's already there to the new device?

**Comment by Marekkon5** [comment, marked as answer]
Hello, you can go to Settings > Advanced > Open data folder and here you can copy the settings.json to another computer. Note: keep 1T closed on the target computer when copying the settings, because if it's open it might overwrite them with the current ones.

**Reply by xonline** [reply]
Thank you!!
